### ISSUE TITLE
Metadata ETL task to Deactivate a Flow

### ISSUE THREAD

**Issue by mattmartin26** [SFDO-Tooling/CumulusCI#3320]
### Feature Type

I'm interested in a new Task or Flow for CumulusCI.

### Does your feature request solve a current problem?

When we deploy a beta package to a client's sandbox org, in order to upgrade to a new beta we have to uninstall the old version. However, there are about a dozen flows that all need to be deactivated before we can do so. 

### Describe the solution you'd like

Just as CCI has a simple task to activate a Flow, we'd love a task that can de-activate a particular Flow as well, so we can create automation to deactivate all of our packaged flows thus expediting the uninstall process. 

### Describe alternatives you've considered

Deactivating via workbench, or via the Salesforce Inspector Chrome plugin. However, this gets us out of our go-to toolset.

**Comment by jstvz** [comment]
Yeah, that's a good idea.

Tracking [W-11588156](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000013wGymYAE/view)

**Comment by mkolodner** [comment]
Adding my vote for this to be built!
I asked for it on TBC: https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000LHM06

**Comment by BrettMN** [comment]
Being able to deactivate a flow through CCI would be awesome and helpful for loading data when setting up new scratch orgs.

**Comment by jstvz** [comment]
Closing as part of a backlog triage pass against v4.10.0 / 5.0.0.dev1.

Reason: feature already implemented

If this is still reproducible against v4.10.0 (the current latest stable) or against v5.0.0.devN on the dev branch, please reopen with a fresh `cci version` block and reproduction steps. We track v5 development on the `dev` branch.
